src/app.ts

```diff
@@ -224,6 +294,23 @@ class FrameworkClient {
 			}
 		}
 	}
+	
+	/**
+	 * Handle the execution of a Slash Command.
+	 * @param interaction The interaction that was received.
+	 * @returns void
+	 */
+	private async handleSlashCommand(interaction: Discord.CommandInteraction) {
+		let commandName = interaction.commandName.toLowerCase();
+		let command:Command = this.slashCommands.find(cmd => cmd.name.toLowerCase() == commandName);
+
+		if(!command) this.error("Somehow this command does not exist..");
```

Comment:
this.error just creates an embed that can be used, it doesn't actually send the error anywhere. Need a log, send to a user, then the function should exit rather than continuing